src/app/public/testing/dropdown/popovers-fixture-dropdown.ts

```diff
@@ -0,0 +1,7 @@
+export interface SkyPopoversFixtureDropdown {
```

Comment:
Does this type deal primarily with the trigger button? What do you think about renaming to `SkyPopoversFixtureDropdownButton` to make it more clear? We can reserve the `SkyPopoversFixtureDropdown` type for something more generic in the future.